packages/terra-base/src/terra-dev-site/doc/base/GlobalBaseStylesExample.jsx

```diff
@@ -14,7 +14,7 @@ const BaseExample = () => (
     <ol>
       <li>Ordered list item</li>
       <li>
-Ordered list item
+        Ordered list item
```

Comment:
Does JSX now ignore these tabs or something? I'm pretty sure all the lack of tabs was to prevent tabs from showing up on the dom. Is that not the case anymore?

Reply:
<img width="455" alt="Screen Shot 2020-06-08 at 11 18 14 AM" src="https://user-images.githubusercontent.com/65959938/84055046-e36e1200-a979-11ea-8fed-1652639e6061.png">

The list looks like this with and without the tabs in the JSX file, so I'm assuming it just ignores the tabs now since there's no change?